The error in analyzegc looks relevant:
https://buildkite.com/julialang/julia-master/builds/25942#01895d3e-b3fc-4405-9273-00114232c1a6/868-1313
```
/cache/build/default-amdci4-3/julialang/julia-master/src/jitlayers.cpp:1220:25: error: Calling potential safepoint as SimpleFunctionCall from function annotated JL_NOTSAFEPOINT [julia.GCChecker]
                assert(!verifyLLVMIR(M));
```

--- a/src/passes.h
+++ b/src/passes.h
@@ -146,8 +146,8 @@ MODULE_MARKER_PASS(BeforeCleanup)
 MODULE_MARKER_PASS(AfterCleanup)
 MODULE_MARKER_PASS(AfterOptimization)
 
-bool verifyLLVMIR(const Module &M);
-bool verifyLLVMIR(const Function &F);
-bool verifyLLVMIR(const Loop &L);
+bool verifyLLVMIR(const Module &M) JL_NOTSAFEPOINT;
+bool verifyLLVMIR(const Function &F) JL_NOTSAFEPOINT;
+bool verifyLLVMIR(const Loop &L) JL_NOTSAFEPOINT;
 
 #endif

--- a/src/pipeline.cpp
+++ b/src/pipeline.cpp
@@ -814,7 +814,7 @@ static llvm::Optional<std::pair<OptimizationLevel, OptimizationOptions>> parseJu
     return {};
 }
 
-bool verifyLLVMIR(const Module &M) {
+bool verifyLLVMIR(const Module &M) JL_NOTSAFEPOINT {
     if (verifyModule(M, &errs())) {
         errs() << "Failed to verify module '" << M.getModuleIdentifier() << "', dumping entire module!\n\n";
         errs() << M << "\n";
@@ -823,7 +823,7 @@ bool verifyLLVMIR(const Module &M) {
     return false;
 }
 
-bool verifyLLVMIR(const Function &F) {
+bool verifyLLVMIR(const Function &F) JL_NOTSAFEPOINT {
     if (verifyFunction(F, &errs())) {
         errs() << "Failed to verify function '" << F.getName() << "', dumping entire module!\n\n";
         errs() << *F.getParent() << "\n";
@@ -832,7 +832,7 @@ bool verifyLLVMIR(const Function &F) {
     return false;
 }
 
-bool verifyLLVMIR(const Loop &L) {
+bool verifyLLVMIR(const Loop &L) JL_NOTSAFEPOINT {
     if (verifyFunction(*L.getHeader()->getParent(), &errs())) {
         errs() << "Failed to verify loop '" << L.getName() << "', dumping entire module!\n\n";
         errs() << *L.getHeader()->getModule() << "\n";

